ctapipe_io_lst/tests/test_stage1.py

```diff
@@ -39,10 +41,8 @@ def test_stage1(tmpdir):
                 'drive_report_path': str(test_drive_report)
             },
             'EventTimeCalculator': {
-                'ucts_t0_dragon': int(run_info['ucts_t0_dragon']),
-                'dragon_counter0': int(run_info['dragon_counter0']),
-                'ucts_t0_tib': int(run_info['ucts_t0_tib']),
-                'tib_counter0': int(run_info['tib_counter0']),
+                'dragon_reference_time': int(run_info['ucts_t0_dragon']),
```

Comment:
Do we still need the int() here?

Reply:
no

Reply:
sorry, yes, we do. Because it is a `numpy.int64` not a python `int`, and the `json` module cannot handle `numpy.int64`. So there is a different reason here for the int call then the reason in lstchain.

Reply:
Without it, you get 'TypeError: Object of type int64 is not JSON serializable'

Reply:
I see, and I guess you can not directly convert to a python int (which is the difference between the two?
) ...

Reply:
I don't understand what you mean by this question, since doing `int(x)` *is the direct way of converting something to a python int*.

The difference is that the pure python `int` is arbitrary length, `np.int64` as the name suggests is restricted to 64 bits.

Reply:
E.g. you can do this with a python int:

```
>>> 2**65
36893488147419103232
```

But doing the same with a numpy int will give you an overflow:
```
>>> np.int64(2)**65
0
```

Reply:
> I don't understand what you mean by this question, since doing `int(x)` _is the direct way of converting something to a python int_.

I mean in the converter function, so that you do not have this further conversion here

Reply:
No, because there we have an astropy table is basically a collection of numpy arrays as columns, so is always using numpy dtypes.

Reply:
ok